Formula/canu.rb

```diff
@@ -0,0 +1,23 @@
+class Canu < Formula
+  desc "Single molecule sequence assembler"
+  homepage "https://canu.readthedocs.org/en/latest/"
+  url "https://github.com/marbl/canu/archive/v1.6.tar.gz"
+  sha256 "470e0ac761d69d1fecab85da810a6474b1e2387d7124290a0e4124d660766498"
+  head "https://github.com/marbl/canu.git"
+  # cite "doi.org/10.1101/gr.215087.116"
```

Comment:
See https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md#citation
(I wrote this text just now.)